scaffolder-templates/github-workflows/advanced-workflow/docs/GithubAdvancedDoc.md

```diff
@@ -0,0 +1,85 @@
+- System: An entity from the catalog.
+
+### Page 2: Build Enviroment
+
+On this page, the user will choose a CI/CD method for building and monitoring their serverless worklow application.
+Upon Choosing 'Tekton with ArgoCD', the user will be prompted to enter some following input parameters.
+
+#### Repository Configuration
+
+- Workflow Namespace: Deployment namespace for workflow applications
+- GitOps Namespace: Deployment namespace for ArgoCD and Tekton resources
+
+#### PostreSQL Configuration:
+
+- PostgreSQL Secret Name: Name of the secret in which the PostgreSQL secrets are stored. Shall be in the same namespace as the workflow
+- PostgreSQL User key from secret: The key name in which the PostgreSQL *user* is stored
+- PostgreSQL Password key from secret: The key name in which the PostgreSQL *password* is stored
+- PostgreSQL K8s Service Name: Name of the service running the PostgreSQL instance
+- PostgreSQL K8s Service Namespace: Namespace of the service running the PostgreSQL instance
+- PostgreSQL Port: Port on which the PostgreSQL instance is running
+
+#### Quay Configuration: 
+
+- Quay Organization Name: The Quay Organization Name of the published workflow
+- Quay Repository Name: A name for the Quay Repository of the published workflow, either existing or to be created
+
+The user can choose an existing Quay repository for pushing the built images, or creating a new one.
+To create a new Quay repository, some additional configuration needs to be made: 
+
+1. The [following dynamic plugin](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-quay) needs to be added to RHDH. 
+To add this plugin, the user can add a new package to the dynamic plugin CR in the rhdh-operator namespace on the cluster
+```
+- disabled: false
+  package: ./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-quay
+```
+2. Create or locate an Organization on Quay
+3. Create an OAuth Application in the organization
+4. Configure 'Create Repositories Access' and grant the OAuth app permission to create new repositories: ![image without spacing](assets/QuayOAuth.png)
+5. Generate an Access Token to use
+
+Upon choosing to create a new one, the user will be prompted for additional input parameters
+
+- Quay AuthToken : Quay organization bearer token used for authorization 
+- Quay Repository Visiblity: Visibility setting for the created repository, either public or private
+- Quay Repository Kind: The created Quay repository kind, either image or application
+- Quay Repository Description: The Quay repository description
+- Quay Base URL: URL of your Quay instance (e.g https://quay.io)
+- Quay Namespace: The Quay namespace of the published workflow. This should be the same as the Quay repository name.
+
+
+### Page 3: Review
+
+The user can review their input parameters and click "Create" to run the template. After completed, links to the created source code repository and GitOps repository will be provided.
```

Comment:
nitpick: 
> After completed, links to the created source code repository and GitOps repository will be provided.

Upon completion, links to the created source code and GitOps repositories will be displayed.

Reply:
If 'None' CI was selected, the gitops link will be broken